packages/cli/src/commands/init/init.js

```diff
@@ -39,93 +39,45 @@ function adjustNameIfUrl(name, cwd) {
   return name;
 }
 
-async function createFromExternalTemplate(
+async function createFromTemplate({
+  projectName,
+  templateName,
+  version,
+  npm,
+}: {
   projectName: string,
   templateName: string,
+  version?: string,
   npm?: boolean,
-) {
-  logger.debug('Initializing new project from external template');
+}) {
+  logger.debug('Initializing new project');
   logger.log(banner);
-
   const Loader = getLoader();
-
   const loader = new Loader({text: 'Downloading template'});
-  loader.start();
-
   const templateSourceDir = fs.mkdtempSync(
     path.join(os.tmpdir(), 'rncli-init-template-'),
   );
-  try {
-    let {uri, name} = await processTemplateName(templateName);
-    await installTemplatePackage(uri, templateSourceDir, npm);
-    loader.succeed();
-    loader.start('Copying template');
-
-    name = adjustNameIfUrl(name, templateSourceDir);
-    const templateConfig = getTemplateConfig(name, templateSourceDir);
-    await copyTemplate(name, templateConfig.templateDir, templateSourceDir);
-
-    loader.succeed();
-    loader.start('Preparing template');
 
-    changePlaceholderInTemplate(projectName, templateConfig.placeholderName);
-
-    loader.succeed();
-    const {postInitScript} = templateConfig;
-    if (postInitScript) {
-      // Leaving trailing space because there may be stdout from the script
-      loader.start('Executing post init script ');
-      await executePostInitScript(name, postInitScript, templateSourceDir);
-      loader.succeed();
-    }
-
-    loader.start('Installing all required dependencies');
-    await PackageManager.installAll({preferYarn: !npm, silent: true});
-    loader.succeed();
-  } catch (e) {
-    loader.fail();
-    throw new Error(e);
-  } finally {
-    fs.removeSync(templateSourceDir);
+  if (version && !semver.gte(version, '0.60.0-rc.0')) {
```

Comment:
it's now possible to try it with RCs

Reply:
Good catch! I hope `gte` will just treat 0.60.0 as higher - I know `npm` had some problems with `^` etc.

Reply:
`gte` - `greater than or equal`